segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/StorageTestBase.java

```diff
@@ -40,13 +42,20 @@
     protected static final int APPENDS_PER_SEGMENT = 10;
     private static final int SEGMENT_COUNT = 4;
 
+    protected StorageMetricsBase metrics = null;
+
     @Override
     protected int getThreadPoolSize() {
         return 5;
     }
 
     //endregion
 
+    static {
```

Comment:
This can be done in setup.

Reply:
A unit test should not affect unit tests that run after it, it should neither leave leftover config, nor leak memory.
I see several other tests invoking this is their setup() methods. If this is interacting with the metrics code badly or is broken, please file an issue explaining it, and/or fix it.